vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -209,7 +209,9 @@ bool FrameworkElementViewManager::UpdateProperty(
           transformMatrix.m43 = static_cast<float>(propertyValue[14].asDouble());
           transformMatrix.m44 = static_cast<float>(propertyValue[15].asDouble());
 
-          ApplyTransformMatrix(element, nodeToUpdate, transformMatrix);
+          element.Loaded([=](auto &&, auto &&) -> auto {
+            ApplyTransformMatrix(element, nodeToUpdate, transformMatrix);
```

Comment:
Well this ever run more than once?

Reply:
Yes this will run whenever the TransformMatrix property is set.  The requirement that the element be in the live tree is going to be problematic due to all the property setters which aren't guaranteed to run after the tree operations.

---
In reply to: [453875881](https://github.com/microsoft/react-native-windows/pull/5499#discussion_r453875881) [](ancestors = 453875881)

Reply:
changed this to differentiate between when the element is loaded or not.